Examples/example_esp_async.py

```diff
@@ -0,0 +1,43 @@
+# The MIT License (MIT)
+# Copyright (c) 2020 Mike Teachman
+# https://opensource.org/licenses/MIT
+
+# example for MicroPython rotary encoder
+#
+# Documentation:
+#   https://github.com/MikeTeachman/micropython-rotary
+
+from rotary_irq_esp import RotaryIRQ
+import uasyncio as aio
+from uasyncio import Event
+from machine import Pin
+
+# Use heartbeat to keep event loop not empty
+async def heartbeat():
+    led = Pin(2, Pin.OUT);
+    while True:
+        led.on();
+        await aio.sleep(0.1);
+        led.off();
+        await aio.sleep(0.8);
```

Comment:
This loop slows down the perceived REPL response of encoder turn.  It is only needed because there is a bug in MicroPython.  See https://github.com/micropython/micropython/issues/5843.
The LED pin is different on boards, so Pin 2 won't toggle an LED on some boards.
My suggestion is to just have a loop like this:

```
# keep the event loop active.
    while True:
        await asyncio.sleep_ms(10)

```

I think this will reduce the time delay to see results at the REPL

Reply:
You are right. I am pretty new to the micropython world. I haven't tried REPL yet.